Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+WORKDIR /sqlite
+RUN apt-get update && apt-get install -y git
```

Comment:
I don't assume you intend this Dockerfile to be distributed as a standalone artifact. Assuming you're fine with this file being stored at the root of this repository, you could just copy the files you need (including git files) and copy those contents into `/sqlite`. Alternatively, you might assume the developer checks out the tag they want before building the docker image. As a new contributor, I'd be surprised that a particular version is enforced by the Dockerfile.

Reply:
> our tool
Which tool?